src/blender_mcp/server.py

```
@@ -266,6 +267,47 @@ def get_object_info(ctx: Context, object_name: str) -> str:
+        # Read the file
+        with open(temp_path, 'rb') as f:
+            image_bytes = f.read()
+        
+        # Delete the temp file
+        os.remove(temp_path)
+        
+        return Image(data=image_bytes, format="png")
+        
+    except Exception as e:
+        logger.error(f"Error capturing screenshot: {str(e)}")
+        raise Exception(f"Screenshot failed: {str(e)}")
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)